pisa/utils/matrix.py

```diff
@@ -0,0 +1,144 @@
+"""
+Utilities for performing some not-so-common matrix tasks.
+"""
+
+import numpy as np
+import scipy.linalg as lin
+
+__all__ = [
+    'is_psd',
+    'fronebius_nearest_psd',
+]
+
+__author__ = 'A. Trettin'
+
+__license__ = '''Copyright (c) 2014-2017, The IceCube Collaboration
```

Comment:
since you presumably just created this file, just list the year 2020 here.

Reply:
ok, I fixed that!